pilot/pkg/security/authz/policy/helper.go

```diff
@@ -125,7 +125,11 @@ func SimpleRole(name string, namespace string, service string) *model.Config {
 }
 
 func BindingTag(name string) string {
-	return fmt.Sprintf("UserFromBinding[%s]", name)
+	return fmt.Sprintf("cluster.local/ns/%s/sa/%s", name, name)
```

Comment:
Why do you need to change this if you already added the new function BindingPrincipal() for using trust domain?